### PR TITLE
Number formatting

### LINKED ISSUE #475: Decimal separators and percentage sign
These are some minor orthography and consistency issues with numbers and percentages. Sometimes the thousands are separated by a space, sometimes not. (Please review attached screenshots)

I'd suggest to use thousands/decimal separators, depending upon the used language. Also, ensure consistent usage of separators accross the website.

- **comma** for thousands `10,000` and **period** for decimal `2.6` in _English_
- **period** for thousands `10.000` and **comma** for decimal `2,6` in _other languages_
- **add a space** between numbers and the percentage sign `0.4 %` or `0,4 %`

1. DecimalSeparator1
----------------------------
2. DecimalSeparator2
----------------------------
3. DecimalSeparator3
----------------------------

And one more thing:
Please check the table **`Structure of confirmed cases`**
If it represents percentages, add the percentage sign.

NumberOrPercentage

### PR DESCRIPTION
**Pull request by VesterDe**
Will hopefully close #475 mostly.

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -49,14 +48,26 @@ function setHighchartsOptions () {
             rangeSelectorZoom: i18next.t("charts.common.zoom"),
             resetZoom: i18next.t("charts.common.resetZoom"),
             resetZoomTitle: i18next.t("charts.common.resetZoomTitle"),
-            thousandsSep: i18next.t("charts.common.thousandsSep"),
-            decimalPoint: i18next.t("charts.common.decimalPoint"),
+            thousandsSep: i18next.separators.decimal,
+            decimalPoint: i18next.separators.group,
```
Comment (stefanb) [inline src/i18n.js:52]: Tole izgleda napak, zamenjano.

Comment (VesterDe) [reply src/i18n.js]: Res je, hvala @stefanb

File: src/visualizations/MunicipalitiesChart.fs [hunk 2]
```diff
@@ -328,17 +328,17 @@ let renderMunicipality (state : State) (municipality : Municipality) =
                         prop.children [
                             Html.div [
                                 prop.className "active"
-                                prop.text (sprintf "%d" (municipality.ActiveCases |> Option.defaultValue 0)) ]
+                                prop.text (sprintf "%s" (I18N.NumberFormat.formatNumber(municipality.ActiveCases |> Option.defaultValue 0))) ]
```
Comment (joahim) [inline src/visualizations/MunicipalitiesChart.fs:331]: You can omit `sprintf "%s"`

File: src/visualizations/MunicipalitiesChart.fs [hunk 3]
```diff
@@ -328,17 +328,17 @@ let renderMunicipality (state : State) (municipality : Municipality) =
...
                             Html.div [
                                 prop.className "total-and-new"
                                 prop.children [
                                     Html.div [
                                         prop.className "total"
-                                        prop.text (sprintf "%d" (municipality.MaxConfirmedCases |> Option.defaultValue 0)) ]
+                                        prop.text (sprintf "%s" (I18N.NumberFormat.formatNumber(municipality.MaxConfirmedCases |> Option.defaultValue 0))) ]
```
Comment (joahim) [inline src/visualizations/MunicipalitiesChart.fs:337]: You can omit `sprintf "%s"`

**Pull request merged**